rero_ils/modules/documents/jsonschemas/documents/document_dissertation-v0.0.1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "dissertation": {
+    "title": "Dissertation or thesis information",
+    "description": "A statement that identifies an academic institution, degree, or year for which a work was presented (MARC 502).",
+    "type": "object",
```

Comment:
Could you make this field repetitive (array)? Gianni just observed that MARC 502 is repetitive.

Reply:
ok